### PR TITLE
Enable SSL setup for database

### PR DESCRIPTION
**Pull request by jsuchome**
~~Requires https://github.com/crowbar/crowbar-openstack/pull/1280~~

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1867,6 +1867,14 @@ function enable_ssl_generic
             fi
             return
         ;;
+        database)
+            if iscloudver 7plus && [[ "$want_database_sql_engine" == "mysql" ]] ; then
+                $p "$a['mysql']['ssl']['enabled']" true
+                $p "$a['mysql']['ssl']['generate_certs']" true
+                $p "$a['mysql']['ssl']['insecure']" true
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: when you append ['mysql'] to $a, you could drop 2 lines and use the generic codepath after the case for mysql - and only do return in the else case

Comment (jsuchome) [reply scripts/qa_crowbarsetup.sh]: Hm, true, but that would be difference to the other services.

Comment (jsuchome) [reply scripts/qa_crowbarsetup.sh]: Also, the generic path does not set "enabled" bit

Comment (bmwiedemann) [reply scripts/qa_crowbarsetup.sh]: would be similar to heat, manila and "*" below.

Comment (bmwiedemann) [reply scripts/qa_crowbarsetup.sh]: See also https://github.com/SUSE-Cloud/automation/pull/2211

**Comment by jsuchome** [comment]
@bmwiedemann @jdsn could you agree on which way this should go? (including https://github.com/SUSE-Cloud/automation/pull/2211)

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (jdsn) [reply scripts/qa_crowbarsetup.sh]: I would recommend to **not** merge #2211 and go with this PR as it is.
Having the config changes explicitly is more clear and easier to understand than hiding it through manipulating `$a`.

**Pull request merged**